content/applications/hr/employees/new_employee.rst

```diff
@@ -79,188 +62,271 @@ Resumé tab
 Resumé
 ~~~~~~
 
-Next, the employee's work history is entered in the :guilabel:`Resumé` tab. Each previous experience
-must be entered individually. Click :guilabel:`Create a New Entry`, and the :guilabel:`Create Resumé
-lines` form appears. Enter the following information, then click the :guilabel:`Save & Close` button
-if there is only one entry to add, or click the :guilabel:`Save & New` button to save the current
-entry and create another line.
+Next, enter the employee's work history in the :guilabel:`Resumé` tab. Each resumé line must be
+entered individually. Click :guilabel:`Create a new entry`, and the :guilabel:`Create Resumé lines`
+form appears. Enter the following information for each entry.
 
 .. image:: new_employee/resume-lines.png
    :align: center
    :alt: Add information for the previous work experience in this form.
 
-- :guilabel:`Name`: Type in the name of the previous work experience.
+- :guilabel:`Title`: Type in the title of the previous work experience.
 - :guilabel:`Type`: From the drop-down menu, select either :guilabel:`Experience`,
-  :guilabel:`Education`, :guilabel:`Internal Certification`, :guilabel:`Internal Training`, or type
-  in a new entry.
+  :guilabel:`Education`, :guilabel:`Side Projects`, :guilabel:`Internal Certification`,
```

Comment:
I checked this, and it never appears in 16.4,  and I tested about 11 different runbots- I got to see it twice in16.2 and once in a 16.3. I am asking the HR experts since I don't understand why it is visible only so few times. I will update this when I have an naswer.

Reply:
Confirmed with the experts: Experience, Education, Side Projects, Internal Certification, an Completed Internal Training are the default options. This has been updated.